packages/mobile/src/Components/Chip/Chip.tsx

```diff
@@ -0,0 +1,3 @@
+import { Chip as PaperChip } from 'react-native-paper';
+
+export default class Chip extends PaperChip {}
```

Comment:
This file name could be index.ts, so when  importing, it would be 
`import Chip from '../Components/Chip';` instead of 
`import Chip from '../Components/Chip/Chip';`